src/bin/pg_autoctl/monitor_pg_init.c

```diff
@@ -47,6 +47,7 @@ GUC monitor_default_settings[] = {
 	{ "log_disconnections", "off" },
 	{ "log_lock_waits", "on" },
 	{ "log_statement", "ddl" },
+	{ "password_encryption", NULL }, /* will be set later */
```

Comment:
```suggestion
	{ "password_encryption", NULL }, /* will be set later based on --auth */
```

Reply:
I'm not so comfortable with having a NULL in there, what about using `"md5"` (or something else) as a placeholder value, or even `"placeholder"` to make it explicit? What could go wrong when we use NULL?

Reply:
because we explicitely handle it and don't write anything in case of other auth methods it shouldn't matter. I had the same thought though. So maybe indeed to be on the safe side for future refactors we should put md5 there.

Reply:
Yeah I was kind of enforcing it with [here](https://github.com/citusdata/pg_auto_failover/pull/383/commits/706c82f8c8023d1ea26c6f3264e623a9e5122b94#diff-4b0fb807d0a2ae6005c10e98bb5a675dR511) with the Assert.

I initially thought of specifying `md5` as a default, but I had decided against it as this default may become inconsistent with PG if ever PG changes the default value to something else. But I don't feel too strongly about that.